src/restElectron.tsx

```diff
@@ -239,31 +239,37 @@ export class RestElectron implements IRest {
             const wallet = masterKey.derive(`m/44'/${this.coinNumber}'/0'/0/0`)
 
             const iv = crypto.randomBytes(16)
-            const key = Buffer.from(utils.blake2bHash(Hwallet.password).buffer)
+            const key = Buffer.from(utils.blake2bHash(Hwallet.password))
```

Comment:
Ah this gets build error. It is better to use `.buffer`

Reply:
Ok thanks.
hycon-ui source is overloaded Buffer.from(): Uint8Array.
But Github source is not.
So I think this problem occurred.